try { throw {[] "foo": {#value "bar", #writable false}}} catch { func (exc) { exc } }
